Update copy-activity-log.md (#69929)
Azure/azure-content: articles/data-factory/copy-activity-log.md

        }

```
> [!NOTE]

> > It isn't possible to provide a user-defined name for the log files which are generated by enabling the `enableCopyActivityLog` property.

Reviewer: The note is useful. What you can do is to update it as "When the 'enableCopyActivityLog' property is set to Enabled, the log file names are system generated."

Author: Thank you @dcstwh for the comment.
I have made the changes as per review comment

> > It is not possible to provide a User defined name to the log files which are generated by enabling the `enableCopyActivityLog` property. `

Reviewer 2: Please use lowercase on user-defined and remove the extra tick at the end of the line. 

```suggestion
> It isn't possible to provide a user-defined name for the log files which are generated by enabling the `enableCopyActivityLog` property.
```
